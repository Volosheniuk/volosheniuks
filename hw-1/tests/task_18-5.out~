125
-125
0
