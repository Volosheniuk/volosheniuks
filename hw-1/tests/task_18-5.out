Syma 1..N =1
